AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumEventDataManager.java

```diff
@@ -205,13 +205,18 @@ public void onCreate(SQLiteDatabase db) {
       db.execSQL("CREATE TABLE IF NOT EXISTS " + EVENT_TABLE_NAME + "(" + COLUMN_DATA +
           " TEXT)");
 
-      // Migrate old data from shared preferences.
-      try {
-        migrateFromSharedPreferences(db);
-      } catch (Throwable t) {
-        Log.e("Cannot move old data from shared preferences to SQLite table.", t);
-        Log.exception(t);
-      }
+      OperationQueue.sharedInstance().addOperation(new Runnable() {
+        @Override
+        public void run() {
+          // Migrate old data from shared preferences.
+          try {
+            migrateFromSharedPreferences(db);
```

Comment:
I think we can remove migrations. It was added when we switched from shared prefs to sqlite, but i dont think anyone is using SDK that old anymore.

Reply:
Done